rclpy/rclpy/signals.py

```diff
@@ -30,6 +30,9 @@ def __del__(self):
         except InvalidHandle:
             # already destroyed
             pass
+        except ValueError:
+            # ValueError: Guard condition was not registered
```

Comment:
Nitpick: just "Guard condition was not registered"?

Reply:
Done in a35aa4e